src/main/java/com/notably/logic/suggestion/handler/DeleteSuggestionArgHandler.java

```diff
@@ -54,13 +59,15 @@ public DeleteSuggestionArgHandler(Model model, CorrectionEngine<AbsolutePath> pa
 
         if (title.isEmpty()) {
             model.setResponseText(RESPONSE_MESSAGE);
+            logger.warning("title is empty");
```

Comment:
This shouldn't be a warning? Since it's a suggestion and it's not really an invalid thing to have empty title